ui/server/src/main/scala/pl/touk/nussknacker/ui/api/SettingsResources.scala

```diff
@@ -6,14 +6,17 @@ import io.circe.generic.JsonCodec
 import pl.touk.nussknacker.engine.ProcessingTypeData
 import pl.touk.nussknacker.engine.ProcessingTypeData.ProcessingType
 import pl.touk.nussknacker.ui.config.FeatureTogglesConfig
-import pl.touk.nussknacker.ui.security.api.LoggedUser
+import pl.touk.nussknacker.ui.security.AuthenticationConfiguration
 
 import scala.concurrent.ExecutionContext
 
-class SettingsResources(config: FeatureTogglesConfig, typeToConfig: Map[ProcessingType, ProcessingTypeData])(implicit ec: ExecutionContext)
-  extends Directives with FailFastCirceSupport with RouteWithUser {
+class SettingsResources(config: FeatureTogglesConfig,
```

Comment:
Question is non-authenticated users can do something with this data? Think they can't..

Reply:
I agree that for unauthorized user here should be minimum setting e.g. `RemoteEnvironmentConfig`, `DeploySettings` for sure shouldn't be visible. We can do it in future but than here should be TODO + very very big WARNING that we left here potential security leak - developer must be warned that if he or she add something here, it will be visible for anyone.

Reply:
Okay. I've added TODO: WARNING now all settings are available for not sign in user. In future we should show only basic settings.